app/views/browse/service-instance.html

```diff
@@ -29,7 +29,10 @@ <h1 class="contains-actions">
                 </ul>
               </div>
               {{serviceInstance | serviceInstanceDisplayName:serviceClasses}}
-              <small class="meta">created <span am-time-ago="serviceInstance.metadata.creationTimestamp"></span></small>
+              <small ng-bind-html="serviceInstance.metadata.name" class="list-row-longname"></small>
```

Comment:
Why is the name `ng-bind-html`?

Reply:
Copied from the overview page. Doesn't need to be. I will update, thanks.